docs/resources/Channel.md

```diff
@@ -1040,3 +1154,117 @@ Adds a recipient to a Group DM using their access token
+
+## Add a user to a thread % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/thread-members/{user.id#DOCS_RESOURCES_USER/user-object}
+
+Adds another user to a thread. Requires the ability to send messages in the thread.  Also requires the thread is not archived. Returns a 204 empty response on success.  Fires a [Thread Members Update](#DOCS_TOPICS_GATEWAY/thread-members-update) Gateway event.
+
+## Leave a thread % DELETE /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/thread-members/@me
+
+Removes the current user from a thread. Returns a 204 empty response on success.  Fires a [Thread Members Update](#DOCS_TOPICS_GATEWAY/thread-members-update) Gateway event.
+
+## Remove a user from a thread % DELETE /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/thread-members/{user.id#DOCS_RESOURCES_USER/user-object}
+
+Removes another user from a thread. Requires the `MANAGE_THREADS` permission or that you are the creator of the thread.  Also requires the thread is not archived. Returns a 204 empty response on success.  Fires a [Thread Members Update](#DOCS_TOPICS_GATEWAY/thread-members-update) Gateway event.
+
+## List Thread Members % GET GET /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/threads-members
+
+Returns array of [thread members](#DOCS_RESOURCES_CHANNEL/thread-member-object) objects that are members of the thread.
+
+> warn
+> This endpoint is restricted according to whether the `GUILD_MEMBERS` [Privileged Intent](#DOCS_TOPICS_GATEWAY/privileged-intents) is enabled for your application.
+
+## List active threads % GET /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/threads/active
+
+Returns all active threads in the channel, including public and private threads.  Threads are ordered by their `id`, in descending order. Requires the `READ_MESSAGE_HISTORY` permission.
+
+###### Response Body
+
+| Field    | Type                                                                            | Description                                                                                  |
+|----------|---------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| threads  | array of [channel](#DOCS_RESOURCES_CHANNEL/channel-object) objects              | the active threads                                                                           |
+| members  | array of [thread members](#DOCS_RESOURCES_CHANNEL/thread-member-object) objects | a thread member object for each returned thread the current user has joined                  |
+| has_more | boolean                                                                         | whether there are potentially additional threads that could be returned on a subsequent call |
```

Comment:
Just, out of curiosity, how does this work exactly? Are we expecting more data when we call the same endpoint? Is it paginated, like `/channels/:id/messages`?

Reply:
Its paginated using the before query param, since the data is in descending order

Reply:
Right, but that's not documented, hence why I asked! :D Then it should be added

Reply:
Oh I'm sorry, I didn't see the context  of this being the active threads endpoint. I assume leaving the has_more in the  response here was an accident, since their is no pagination (based on the removal of it in a prior commit)

Reply:
Oh I'm sorry, I didn't see the context of this being the active threads endpoint. I assume leaving the has_more in the response here was an accident, since their is no pagination (based on the removal of it in a prior commit)

Reply:
Oh yeah, that's fair it doesn't tell you want to do lol, whoops
But yeah, it's in descending order so you'd take the lowest id or archive_timestamp and pass that as before

Reply:
Wait, so this endpoint can be paginated by ID *or* archive timestamp? 👀 
That's sweet if that's the case

Reply:
No, each endpoint can only be paginated by one of them